debug/debug-network.md

```diff
@@ -1,14 +1,14 @@
 # Debugging a WordPress Network
 
-## Debugging a WordPress Network {#debugging-a-wordpress-network}
+## Debugging a WordPress Network
 
-If you have reached this page, chances are you have received an error in your [WordPress network](https://wordpress.org/documentation/articles/glossary#network). This failure occurs when WordPress cannot find one or more of the global tables for the network in the [database](https://codex.wordpress.org/Database_Description).
+If you have reached this page, chances are you have received an error in your [WordPress network](https://wordpress.org/documentation/article/multisite-network-administration/). This failure occurs when WordPress cannot find one or more of the global tables for the network in the [database](https://codex.wordpress.org/Database_Description).
 
-On some shared webhosts, the host has disabled the functionality from running. It is always best to check with your webhost **before** [creating a network](https://developer.wordpress.org/advanced-administration/wordpress/wp-config/) to make sure your webhost account fulfills the technical requirements.
+On some shared webhosts, the host has disabled the functionality from running. It is always best to check with your webhost **before** [creating a network](https://github.com/WordPress/Advanced-administration-handbook/blob/main/multisite/create-network.md) to make sure your webhost account fulfills the technical requirements.
```

Comment:
The handbook will be transferred over to wp.org and the links should still point there.

Reply:
The future link for “prepare a network” is:
https://developer.wordpress.org/advanced-administration/multisite/prepare-network/

The future link for “create a network” is:
https://developer.wordpress.org/advanced-administration/multisite/create-network/